rageval/metrics/_answer_bleu.py

```diff
@@ -0,0 +1,162 @@
+from dataclasses import dataclass
+from typing import List, Tuple
+
+import datasets
+from datasets import Dataset
+import re
+
+from rageval.metrics import Metric, add_attribute
+
+
+_DESCRIPTION = """\
+BLEU (Bilingual Evaluation Understudy) is an algorithm for evaluating the quality of text which has been machine-translated from one natural language to another.
+Scores are calculated for individual translated segments—generally sentences—by comparing them with a set of good quality reference translations.
```

Comment:
PROOFREAD THE SENTENCES!!!

Scores are calculated by comparing individual translated segments, e.g., sentences, with a set of high-quality reference translations.